third-party/realdds/include/realdds/topics/image/image-msg.h

```diff
@@ -34,14 +43,32 @@ class image
 public:
     using type = raw::device::imagePubSubType;
 
+    image() = default;
     image( const raw::device::image & );
+    image( const image & );
+    image( image && );
+
+    image & operator=( image && rhs );
+    image & operator=( const raw::device::image & );
```

Comment:
Should be `&&` but I don't see why you'd need it...?

Reply:
The line `*output = raw_data` in `take_next` use this. Before it used the constructor from `raw::device::image` removed in the previous conversation.
With the constructor it built a temporary object and then moved the contents to `output`, now it avoids the temporary

Reply:
Cannot use default because it's converting from `raw::device:image`. Default is only for move from same type.

Deleted all other construct/copy/move functions.